lib/resources/Account.js

```diff
@@ -2,12 +2,40 @@
 
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
+var utils = require('../utils');
 
 module.exports = StripeResource.extend({
-  path: 'account',
+  // Since path can either be `account` or `accounts`, support both through stripeMethod path
```

Comment:
I think the people who care about their own account is notoriously small anyway, it seems like 100% a Connect use case. But we will over time consider exposing this to managing your own account (i.e. you can update your own account), so consolidation seems to make sense to me. Happy to IRL though, I'll find you.